index.js

```diff
@@ -1,6 +1,7 @@
 import { CompanyTypes, createScraper } from "israeli-bank-scrapers";
+import { sendTelegramMessage } from "./telegram.js";
```

Comment:
You didn't include those modules.

Reply:
What do you mean?

Reply:
There's no `telegram.js` and `discord.js` in the PR.